lib/fill/index.js

```diff
@@ -4,6 +4,6 @@ export default function(val, start, end) {
   return function(arr) {
     return val !== undefined
       ? toArray(arr).fill(val, start || 0, end || arr.length)
-      : arr
+      : arr.slice()
```

Comment:
Inconsistent. “toarray”, or “slice” should be used in both cases.
I think, maybe we should write it to variable
var newArray = toArray(arr)
and use use further

Reply:
I will remove `toArray(arr)` as it's useful when `arr` is functions' `arguments`